.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: android
+
+before_install:
+    - echo yes | android update sdk --filter extra-android-support --no-ui --force > /dev/null
+    - echo yes | android update sdk --filter extra-android-m2repository --no-ui --force > /dev/null
+
+android:
+  components:
+    - build-tools-23.0.2
+    - android-23
+    - extra-android-m2repository
+
+  licenses:
+    - android-sdk-license-.+
+
+script:
+  ./gradlew checkstyle build
```

Comment:
Please add a "\n" at the end of the file because a line is always followed by a linebreak